src/hypothesis/core.py

```diff
@@ -92,15 +92,10 @@ def reify_and_execute(
     print_example=False,
     is_final=False,
 ):
-    from hypothesis.strategies import random_module
-
     def run(data):
-        from hypothesis.control import note
-
         with BuildContext(data, is_final=is_final):
-            seed = data.draw(random_module()).seed
-            if seed != 0:
-                note('random.seed(%d)' % (seed,))
+            import random as rnd_module
+            rnd_module.seed(0)
```

Comment:
I think I’m missing something – won’t this always re-seed the random module to 0, even if the user has explicitly seeded it with a different value?